packages/components/organisms/f-checkout/src/services/mapper.js

```diff
@@ -83,4 +89,23 @@ const mapAnalyticsName = field => (analyticFieldNameMapper[field] || field);
  */
 const mapAnalyticsNames = fields => fields.map(mapAnalyticsName).sort();
 
-export { mapUpdateCheckoutRequest, mapAnalyticsName, mapAnalyticsNames };
+/**
+ * Updates passed issues to match the expected analytics error requirements.
+ */
+const mapAnalyticsError = issue => {
+    let errors;
+
+    Object.entries(updateCheckoutErrors).forEach(entry => {
+        const [errorName, errorList] = entry;
+        if (errorList.includes(issue)) {
+            errors = errorName;
+        }
+    });
+
+    return errors;
+};
```

Comment:
Let's pair on this one since I'm not entirely sure I understand what we're doing 😬